openfisca_core/simulations/simulation_builder.py

```diff
@@ -145,17 +207,17 @@ def build_from_entities(self, tax_benefit_system, input_dict):
 
         return simulation
 
-    def build_from_variables(self, tax_benefit_system, input_dict):
+    def build_from_variables(
+        self, tax_benefit_system: TaxBenefitSystem, input_dict: AbbrParams
+    ) -> Simulation:
         """
         Build a simulation from a Python dict ``input_dict`` describing variables values without expliciting entities.
 
         This method uses :any:`build_default_simulation` to infer an entity structure
 
         Example:
+            >>> {'salary': {'2016-10': 12000}}
```

Comment:
Nitpick: why remove the function call from the example ? As I understand it example in docstrings are executable and this is often helpful (if somewhat redundant with a good test suite).

Reply:
The thing is that they do not work, and the linter was complaining. For this PR, we could actually leave it as it was. Ideally, however, I've been fixing doctests and adding them to the test suite, and I'd definitely do that in another PR.